app/modules/asset_groups/models.py

```diff
@@ -382,14 +401,16 @@ class AssetGroupSighting(db.Model, HoustonModel):
         default=AssetGroupSightingStage.unknown,
         nullable=False,
     )
-    owner_guid = db.Column(
```

Comment:
This kept tripping me up so it was time to change it. The owner of a thing is a user, this is a reference to the AssetGroup that has this AssetGroupSighting in it